### ISSUE TITLE
Government Controlled Claims

### ISSUE THREAD

**Issue by Deantwo** [StrangeLoopGames/EcoSuggestions#624]
Currently we get claim papers from learning skill scrolls. This has the disadvantages of giving players way more claim than they need, and it gives the government no way control how many claims a player can have.

I suggest that claim papers simply be removed. Allow players to claim as many plots as they want with the claim tool. This would encourage players to then propose laws or government systems for controlling the number of claims each player can have.

There are however also requests for wanting to somehow keep some way of making claim papers an item so it can be traded and sold. So maybe creation of claim papers can be moved to the real estate desk or something, so the government can print claim permits that can be sold at stores.

**Comment by elfl0rd** [comment]
I think that "claim paper" should be made by a separate profession.
(e.g. craft claim papers from paper and lot of gold.)
In addition, it is a good idea to make this profession available only to members of the government.

**Comment by Deantwo** [comment]
@alexarthur94, Too many things use property to determine ownership, such as crafting stations, houses, stores. Without that it would be a nightmare to prevent greifing and theft.

It is true that property was less controlled and less enforced back in the day, but thst didn't mean property didn't exist. Not to mention this is a game, so total anacey is likey a bad idea, nothing stops you try trying to do it though.

As I appemted to suggest in the opebing post. Instead of having no properties, we simply aren't limited in how much we can claim before a government or laws are agreed on.
This could however also result in greifing, so likely unlimited claiming would be restricted by the default Citizenship law.
But the government also need to somehow gain the ability to unclaim property of other players. So admins aren't required to clean up messes made by bad laws.

On the topic of allowing government to unclaim people's claims at will, maybe the government elections would be taken more serious. You wouldn't vote a troll that plan to unclaim your property and steal your stuff into office.

**Comment by alexarthur94** [comment]
I agree that the government should have the ability to unclaim; real governments do unless their own laws prevent them. 

I'm not saying there shouldn't be claim papers, but I do feel like they should only be made by the government except for a small customizable number to start with, unless a given server decides on a different system. In lieu of the government creating them, that would force players to create an active government. 

In the real world past, property was recognized and enforced, but that was something that people had to agree on. If one individual simply went somewhere and expected no one to cross the lines he specified, that wouldn't have worked unless no one ever came or it was already a social custom. 

Personally, I feel like allowing people unlimited claims until the government does something could actually be a worse scenario.

**Comment by Deantwo** [comment]
Then simply make a new default law that limit claiming to 10, or something. I am not saying I want all players to run around and claim as much land as possible before a law can be passed to prevent it. But the limit should then be easy to modify.

A default law like this could work:
```
When attempting to claim property (all)
    if (is a (demographic) Citizen)
        if (your number of claims < 10)
            allow
        otherwise
            prevent
    otherwise
        if (your number of claims < 5)
            allow
        otherwise
            prevent
```

**Comment by alexarthur94** [comment]
That law is a useful suggestion, Deantwo. Thanks!
